tests/js/server/recovery/applier-restart.js

```diff
@@ -41,6 +41,7 @@ function runSetup () {
   replication.applier.properties({ endpoint: "tcp://ignoreme.arangodb.com:9999", autoStart: true });
 
   internal.debugTerminate('crashing server');
+  return 0;
```

Comment:
Can this line ever be reached after a `debugTerminate()` call?

Reply:
it should not. however, the invoking boiler plate will return this.